node_modules/oae-tests/lib/util.js

```diff
@@ -256,18 +257,23 @@ var generateTestUsers = module.exports.generateTestUsers = function(restCtx, tot
                 return callback(err);
             }
 
-            _createdUsers.push({
-                'user': user,
-                'restContext': new RestContext(restCtx.host, {
-                    'hostHeader': restCtx.hostHeader,
-                    'username': username,
-                    'userPassword': 'password',
-                    'strictSSL': restCtx.strictSSL
-                })
-            });
+            // Manually verify the user their email address
+            PrincipalsDAO.setEmailAddress(user, email.toLowerCase(), null, function(err, user) {
```

Comment:
`setEmailAddress` takes care of multiple things such as:
 - Setting `email` in the `Principals` CF
 - Adding the reverse email2user mapping
 - Removing the old email2user mapping (if the user has an updated email address)
 - Removing any email tokens associated with that user